blueprints/showbie-tooling/index.js

```diff
@@ -10,18 +11,24 @@ module.exports = {
         { name: 'ember-cli-eslint', target: '~3.0.0' },
         { name: 'ember-cli-code-coverage', target: '~0.3.4' }
       ]
-    })
-    .then(() => {
+    }).then(() => {
       return this.addPackagesToProject([
         { name: 'eslint-plugin-ember-suave', target: '~1.0.0' },
-        { name: 'lodash.samplesize', target: '^4.2.0' }
+        { name: 'lodash.samplesize', target: '^4.2.0' },
+        { name: 'stylelint', target: '~8.2.0' },
+        {
+          name: 'stylelint-config-property-sort-order-smacss',
+          target: '~2.0.0'
+        },
+        { name: 'stylelint-config-sass-guidelines', target: '~3.1.0' },
+        { name: 'stylelint-order', target: '^0.7.0' },
```

Comment:
Might as well lock with with `~` like the rest of the deps. `^` is the devil.

Reply:
I believe in versions without a major version number, `^` acts like `~`. [[reference](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004)]

Reply:
Fair enough. I'm just thinking more about consistency. And what happens when 1.0.0 comes out and someone runs `yarn upgrade stylelint-order`? Does it bump the major version?